src/main/java/frc/robot/AutoPaths.java

```diff
@@ -0,0 +1,394 @@
+package frc.robot;
+
+import choreo.auto.AutoRoutine;
+import choreo.auto.AutoTrajectory;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.subsystems.drivetrain.Drivetrain;
+
+public class AutoPaths {
+    private final Drivetrain drivetrain;
+    private final SuperStructure superStructure;
+    private final choreo.auto.AutoFactory autoFactory;
+
+    public AutoPaths(
+            Drivetrain drivetrain,
+            SuperStructure superStructure, choreo.auto.AutoFactory autoFactory) {
+        this.drivetrain =drivetrain;
+        this.superStructure =superStructure;
+        this.autoFactory =autoFactory;
+    }
+
+    public AutoRoutine facePlantG() {
+        AutoRoutine routine = autoFactory.newRoutine("FacePlantG");
+
+        AutoTrajectory facePlantGTraj = routine.trajectory("FacePlantG");
+
+        routine.active().onTrue(
+                Commands.sequence(
+                        facePlantGTraj.resetOdometry(),
+                        Commands.waitSeconds(2).andThen(
+                                Commands.parallel(
+                                        facePlantGTraj.cmd(),
+                                        superStructure.followPositions(() -> RobotMode.CORAL_LEVEL_4))
+                        )
+                )
+        );
+        return routine;
+    }
+
+    public AutoRoutine facePlantH() {
+        AutoRoutine routine = autoFactory.newRoutine("FacePlantH");
+
+        AutoTrajectory facePlantHTraj = routine.trajectory("FacePlantH");
+
+        routine.active().onTrue(
+                Commands.sequence(
+                        facePlantHTraj.resetOdometry(),
+                        Commands.waitSeconds(2).andThen(Commands.parallel(
+                                facePlantHTraj.cmd(), superStructure.followPositions(() -> RobotMode.CORAL_LEVEL_4))
+                        )
+                )
+        );
+
+        return routine;
+    }
+
+    public AutoRoutine twoCoralDC() {
+        AutoRoutine routine = autoFactory.newRoutine("TwoCoralDC");
+
+        AutoTrajectory Start = routine.trajectory("PStart-D");
+        AutoTrajectory DWCS = routine.trajectory("D-WCS");
+        AutoTrajectory WCSC = routine.trajectory("WCS-C");
```

Comment:
Can we change the capitalization of these to be the correct capitalization for variables.

We'll want to do this in all auto methods.